ls1intum/Artemis
# [Bugfix] Fix missing build failed annotation

### Checklist

### Motivation and Context


When generating a build failed, the erroneous line was not annotated with a red x. This PR fixes this.

### Description

- Adapted the regex for the error 
### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration
3. Create Programming Exercise and participate
4. Remove e.g. the `{` in `public class BubbleSort {` in the 'BubbleSort.java
5. Check that the red x icon is displayed as annotation in the line of the change

### Test Coverage



Nothing changed.

### Screenshots


![image](https://user-images.githubusercontent.com/41108487/106654213-1fed1e80-6598-11eb-918e-b1ee2ae6741f.png)
